lms/models/grouping.py

```diff
@@ -177,35 +177,55 @@ def get_group_sets(self) -> List[dict]:
         """Get this course's available group sets."""
         return self.extra.get("group_sets", [])
 
+    def get_mapped_page_id(self, page_id):
```

Comment:
New methods to store the mapping between pages

this will by a dictionary of old_page_id -> new_page_id

Same mechanism as for files and groups.